src/libPMacc/include/algorithms/math/doubleMath/floatingPoint.tpp

```diff
@@ -60,6 +72,29 @@ struct Float2int_rd<double>
     }
 };
 
+template<>
+struct Float2int_rn<double>
+{
+    typedef int result;
+
+    HDINLINE result operator( )(double value)
+    {
+#if __CUDA_ARCH__
+        return ::__double2int_rn( value );
+#else
+        if(value < 0.0)
+            return -(*this)(-value);
+        static const result epsilon = 1e-7;
```

Comment:
alternatively, [std::numeric_limits::epsilon](http://en.cppreference.com/w/cpp/types/numeric_limits/epsilon) is the right choice.